common/config/rush/common-versions.json

```diff
@@ -83,23 +83,29 @@
       "~0.6.1" // API Extractor is using an older version of source-map because newer versions are async
     ],
 
-    "webpack": ["~5.35.1"],
+    "webpack": ["~4.44.2"],
 
     // Use two different versions of @types/webpack-dev-server to allow pnpmfile.js to bring in
     // two different versions of the webpack typings
     "@types/webpack-dev-server": ["3.11.3"],
 
-    "webpack-dev-server": [
-      "~3.11.0" // heft-plugins/heft-webpack4-plugin is using an older version of webpack-dev-server for compatibility
-    ],
-
-    "html-webpack-plugin": [
-      "~4.5.0" // Some test projects are still on Webpack 4 because some of our plugins don't support Webpack 5 yet
-    ],
+    // heft-plugins/heft-webpack4-plugin is using an older version of webpack-dev-server for compatibility
+    "webpack-dev-server": ["~3.11.0"],
```

Comment:
Should this move down to the new `    // --- For Webpack 4 projects ----` section?

Reply:
fixed